cloud_experts_tutorials/rosa-mobb-aws-cloudwatch-publish-metrics.adoc

```diff
@@ -0,0 +1,173 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-cloudwatch-publish-metrics"]
+= Tutorial: Using the AWS CloudWatch agent to publish metrics to CloudWatch in ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-cloudwatch-publish-metrics
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-19
+//---
+//date: '2021-10-04'
+//title: Using the AWS Cloud Watch agent to publish metrics to CloudWatch in ROSA
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Kevin Collins
+//---
+
+Use the Amazon Web Services (AWS) CloudWatch agent to scrape Prometheus endpoints and publish metrics to CloudWatch in a {product-title} (ROSA) cluster.
+
+This tutorial pulls from the AWS documentation to install the CloudWatch agent to Kubernetes, publish metrics for the Kubernetes API server, and provide a simple dashboard to view the results.
+
+[IMPORTANT]
+====
+The AWS CloudWatch agent does link:https://github.com/aws/amazon-cloudwatch-agent/issues/187[not support] pulling all metrics from the Prometheus federated endpoint.
+====
+
+.Prerequisites
+
+* The link:https://aws.amazon.com/cli/[AWS CLI]
+* The link:https://stedolan.github.io/jq/[`jq` command]
+* A ROSA cluster
+
+== Preparing your AWS account
+. Turn off AWS CLI paging:
++
+[source,terminal]
+----
+$ export AWS_PAGER=""
+----
+
+. Set the following environment variables, changing them to suit your environment:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=metrics
+$ export CLUSTER_REGION=us-east-2
+$ export SCRATCH_DIR=/tmp/scratch
+$ mkdir -p $SCRATCH_DIR
+----
+
+. Create an AWS IAM user for CloudWatch:
++
+[source,terminal]
+----
+$ aws iam create-user \
+--user-name $CLUSTER_NAME-cloud-watch \ 
+> $SCRATCH_DIR/aws-user.json
```

Comment:
Was the `>` included or an accident? I don't recognize what it would do in this command.